src/terms/kinetic.jl

```diff
@@ -40,3 +44,44 @@ end
 
     (E=E, ops=ops)
 end
+
+"""
+Blow-up function as proposed in [REF paper Cancès, Hassan, Vidal to be submitted]
+"""
+function Blowup_function_CHV()
```

Comment:
Be careful. This code is generally not typestable. E.g. $x$ could be a Float32 or an interval or a dual number.

Reply:
Also it feels weird to have this function returning a function. Why is this needed? Can't the outer be merged into the inner?

Reply:
ok I have a better idea, see my comment below.